src/gfx/context.cpp

```diff
@@ -431,7 +448,7 @@ void Context::requestDevice() {
   deviceDesc.nextInChain = &deviceExtras.chain;
 #endif
 
-  getLogger()->debug("Requesting wgpu device");
+  SPDLOG_LOGGER_DEBUG(getLogger(), "Requesting wgpu device");
```

Comment:
Still wondering why you didn't use existing macros here

Reply:
I want to send the graphics logs to it's own logger so I can filter them out / tag them / etc.